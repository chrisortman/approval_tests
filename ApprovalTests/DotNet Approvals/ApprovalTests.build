<?xml version="1.0"?>
<project name="ApprovalTests" default="nightlyBuild">
	<property name="base" value="${directory::get-current-directory()}" />
	<property name="properites" value="ApprovalTests.properties" />
	<include buildfile="${properites}" if="${file::exists(properites)}"/>
		
	<loadtasks assembly="${base}/Common/nantcontrib-0.85/bin/NAnt.Contrib.Tasks.dll" />
	
	<target name="nightlyBuild" depends="build,test,package, sourceforge">
	</target>
	
	<target name="build">
		<msbuild project="${base}/ApprovalTests.sln"/>
	</target>

	<target name="test">
		<nunit2>
			<formatter type="Xml" usefile="true" extension=".xml" outputdir="${base}/results" />
			<test assemblyname ="${base}/ApprovalTests.Tests/bin/debug/ApprovalTests.Tests.dll" />
		</nunit2>
	</target>

	<target name="package">
		<mkdir dir="${base}/results"/>
		<zip zipfile="${base}/results/ApprovalTests.zip">
			<fileset basedir="${base}/ApprovalTests/bin/debug/">
				<include name="ApprovalTests.dll" />
			</fileset>
		</zip>
	</target>

	<target name="sourceforge">
		<scp program="${base}/build/pscp.exe" file="${base}/results/ApprovalTests.zip" server="frs.sourceforge.net" path="${uploadPath}" user="${username}" options="-pw ${password}"/>
	</target>
</project>

