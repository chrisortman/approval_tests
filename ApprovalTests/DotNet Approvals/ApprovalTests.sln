Microsoft Visual Studio Solution File, Format Version 10.00
# Visual Studio 2008
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ApprovalTests", "ApprovalTests\ApprovalTests.csproj", "{F70C9A76-F797-4167-8178-14821F8634BC}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ApprovalTests.Tests", "ApprovalTests.Tests\ApprovalTests.Tests.csproj", "{7B8D265D-35BB-4817-B074-F5056E5A9B4E}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ApiReporter", "ApiReporter\ApiReporter.csproj", "{5F11F7C5-A3E9-4276-9883-09BC41B51444}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ApiReporter.Tests", "ApiReporter.Tests\ApiReporter.Tests.csproj", "{67B7DE06-9DB5-4CBB-A18C-669211FF694E}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{CC679657-AF36-470E-9976-BB00E6792F32}"
	ProjectSection(SolutionItems) = preProject
		ApprovalTests.build = ApprovalTests.build
		ApprovalTests.vsmdi = ApprovalTests.vsmdi
		LocalTestRun.testrunconfig = LocalTestRun.testrunconfig
	EndProjectSection
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ApprovalTests.Extensions", "ApprovalTests.Extensions\ApprovalTests.Extensions.csproj", "{41D87560-5BD3-4AD4-91E4-104D954D2A20}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ApprovalTests.Extensions.Tests", "ApprovalTests.Extensions.Tests\ApprovalTests.Extensions.Tests.csproj", "{7F276069-5FA5-466E-9038-4B61EAACE866}"
EndProject
Global
	GlobalSection(SolutionConfigurationPlatforms) = preSolution
		Debug|Any CPU = Debug|Any CPU
		Release|Any CPU = Release|Any CPU
	EndGlobalSection
	GlobalSection(ProjectConfigurationPlatforms) = postSolution
		{F70C9A76-F797-4167-8178-14821F8634BC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{F70C9A76-F797-4167-8178-14821F8634BC}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{F70C9A76-F797-4167-8178-14821F8634BC}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{F70C9A76-F797-4167-8178-14821F8634BC}.Release|Any CPU.Build.0 = Release|Any CPU
		{7B8D265D-35BB-4817-B074-F5056E5A9B4E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{7B8D265D-35BB-4817-B074-F5056E5A9B4E}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{7B8D265D-35BB-4817-B074-F5056E5A9B4E}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{7B8D265D-35BB-4817-B074-F5056E5A9B4E}.Release|Any CPU.Build.0 = Release|Any CPU
		{5F11F7C5-A3E9-4276-9883-09BC41B51444}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{5F11F7C5-A3E9-4276-9883-09BC41B51444}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{5F11F7C5-A3E9-4276-9883-09BC41B51444}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{5F11F7C5-A3E9-4276-9883-09BC41B51444}.Release|Any CPU.Build.0 = Release|Any CPU
		{67B7DE06-9DB5-4CBB-A18C-669211FF694E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{67B7DE06-9DB5-4CBB-A18C-669211FF694E}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{67B7DE06-9DB5-4CBB-A18C-669211FF694E}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{67B7DE06-9DB5-4CBB-A18C-669211FF694E}.Release|Any CPU.Build.0 = Release|Any CPU
		{41D87560-5BD3-4AD4-91E4-104D954D2A20}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{41D87560-5BD3-4AD4-91E4-104D954D2A20}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{41D87560-5BD3-4AD4-91E4-104D954D2A20}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{41D87560-5BD3-4AD4-91E4-104D954D2A20}.Release|Any CPU.Build.0 = Release|Any CPU
		{7F276069-5FA5-466E-9038-4B61EAACE866}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{7F276069-5FA5-466E-9038-4B61EAACE866}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{7F276069-5FA5-466E-9038-4B61EAACE866}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{7F276069-5FA5-466E-9038-4B61EAACE866}.Release|Any CPU.Build.0 = Release|Any CPU
	EndGlobalSection
	GlobalSection(SolutionProperties) = preSolution
		HideSolutionNode = FALSE
	EndGlobalSection
EndGlobal
