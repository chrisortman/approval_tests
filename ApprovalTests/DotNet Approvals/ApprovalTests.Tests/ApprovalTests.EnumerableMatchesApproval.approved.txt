a[0] = abc
a[1] = 123
a[2] = !@#
